### PR TITLE
Ml on gcp keras for text classification

### PR DESCRIPTION
**Pull request by BenoitDherin**
This PR contains to labs:

- automl_for_text_classification
- keras_for_text_classification

### CODE REVIEW

File: courses/machine_learning/text_classification/automl_for_text_classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,499 @@
+{
```
Comment (munnm) [inline courses/machine_learning/text_classification/automl_for_text_classification.ipynb]: <p>typo: "..model we have just train." to "model we have just trained."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (munnm) [inline courses/machine_learning/text_classification/automl_for_text_classification.ipynb]: <p>You have a comment here about repeatable splits for train/eval/test but it doesn't seem to show up in the queries? Also (and instead), might be mentioning that you can pass to AutoML the TRAIN/VALID/TEST catagory for each training example or just let AutoML figure it out. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (munnm) [inline courses/machine_learning/text_classification/automl_for_text_classification.ipynb]: <p>typo: "articles"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (munnm) [inline courses/machine_learning/text_classification/automl_for_text_classification.ipynb]: <p>typo in SAMPLE_DATASET_NAME. Should be "titles_sample.csv"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (munnm) [inline courses/machine_learning/text_classification/automl_for_text_classification.ipynb]: <p>typo: "titles_sample.csv"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

File: courses/machine_learning/text_classification/keras_for_text_classification.ipynb [hunk 2]
```diff
@@ -0,0 +1,1174 @@
+{
```
Comment (munnm) [inline courses/machine_learning/text_classification/keras_for_text_classification.ipynb]: <p>LGTM</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

File: courses/machine_learning/text_classification/automl_for_text_classification.ipynb (hunk 1 above)
Comment (BenoitDherin) [reply courses/machine_learning/text_classification/automl_for_text_classification.ipynb]: <p>Done!</p>

Comment (BenoitDherin) [reply courses/machine_learning/text_classification/automl_for_text_classification.ipynb]: <p>Done!</p>

Comment (BenoitDherin) [reply courses/machine_learning/text_classification/automl_for_text_classification.ipynb]: <p>Done!</p>

Comment (BenoitDherin) [reply courses/machine_learning/text_classification/automl_for_text_classification.ipynb]: <p>Done!</p>

Comment (BenoitDherin) [reply courses/machine_learning/text_classification/automl_for_text_classification.ipynb]: <p>Done!</p>

**Pull request merged**